e2e/__tests__/__snapshots__/detectOpenHandles.ts.snap

```diff
@@ -13,7 +13,13 @@ This usually means that there are asynchronous operations that weren't stopped i
 `;
 
 exports[`prints out info about open handlers 1`] = `
-Jest has detected the following 1 open handle potentially keeping Jest from exiting:
+Jest has detected the following 2 open handles potentially keeping Jest from exiting:
+
+  ●  DNSCHANNEL
+
+      at dns.js:333:23
+      at Object.listen (server.js:9:5)
```

Comment:
this is unexpected?

Reply:
I forgot to mention that when I run `yarn test`, two tests are failing and this is one of them.

Reply:
Reverted this commit from 5 hours ago, and updated the `package` and `babel` configuration files in the `enzyme` and `react-testing-library` examples -- this unexpected change does not occur.